### DISCUSSION TITLE
How to get the simulation result from Maxwell3D

### DISCUSSION THREAD

**Discussion by QY7** [ansys/pyaedt#3180] [Q&A] [answered]
Hi, everyone. I'm new to pyaedt, and I want to use python to retrieve the simulation report from AnsysEM  Maxwell3D project. For example, I want to get the B field in a geometry. How to do this? I did not found any detailed documentation about this. I'll be appreciated if you help me solve this problem, Or maybe you can provide me with where to find the solution. Thanks

**Comment by Leo-TNKuo** [comment, marked as answer]
Hi Qy7,

You may refer to the link below. Although it is in Chinese, you can use Google translate to your language.
[Field Calculator.](https://eeman.gitbook.io/fields-calculator-of-maxwell/si-zi-dong-hua-ying-yong/ci-tong-mi-du-zi-dong-ji-suan-jia-ru-pyaedt-zhong-shi-yong)

**Reply by QY7** [reply]
Thank you so much for your information, this is useful for me !
